README.md

```diff
@@ -907,42 +907,25 @@ This allows multiple pipelines to use the same code for share tools and gives a
 
 The nf-core DSL2 modules repository is at <https://github.com/nf-core/modules>
 
+### Running the module commands
+
+The `nf-core/tools` package contains a suite of commands for working DSL2 modules in your pipeline. The modules commands use `git` to interface
+with the `nf-core/modules` repository. Therefore you need to have have `git` installed to use the modules commands. You can then use the
+the modules commands for a variety of tasks such as creating new DSL2 modules, viewing and getting metadata about the modules in your pipeline,
+and installing and updating modules from a remote git repository.
+
 ### Custom remote modules
 
 The modules supercommand comes with two flags for specifying a custom remote:
 
-- `--github-repository <github repo>`: Specify the repository from which the modules should be fetched. Defaults to `nf-core/modules`.
-- `--branch <branch name>`: Specify the branch from which the modules shoudl be fetched. Defaults to `master`.
+- `--git-remote <git remote url>`: Specify the repository from which the modules should be fetched as a git URL. Defaults to the github repository of `nf-core/modules`.
+- `--branch <branch name>`: Specify the branch from which the modules should be fetched. Defaults to the default branch of your repository
 
 Note that a custom remote must follow a similar directory structure to that of `nf-core/moduleś` for the `nf-core modules` commands to work properly.
 
-### Private remote modules
```

Comment:
Let's keep this heading and just say that it should work if local `git` is configured with access.